dwcj-engine/src/main/java/org/dwcj/controls/AbstractDwcControl.java

```diff
@@ -236,7 +236,7 @@ public AbstractDwcControl removeStyle(String property) {
     if (ctrl != null) {
       try {
         // Current BBj implementation does not have a remove style method
-        ctrl.unsetStyle(property);
```

Comment:
Why you changed this? This method is to remove a style property

Reply:
I was assuming the new removeClass method was supposed to replace this? If not, my mistake and I can revert it!

Reply:
It is no. Please revert this change and test again. You should be able to remove built-in styles